_posts/2013-03-12-New-API-Blueprint-Format-Roll-Out.md

```diff
@@ -16,12 +16,14 @@ First I would like to thank our awesome users who contributed to the New Format
 
 In the process we have made quite a few changes to the Format. For example, to improve both readability and the process of writing we have decided to leave the "header marks (#) forest" in favor of utilizing "much cleaner" Markdown lists, like so: 
 
-	# GET /message_of_the_day
-	Retrieves message of the day.
-	
-	+ Response 200 (application/json)
-			
-		{ "message" : "Hello World" }
+```apiblueprint
```

Comment:
The rouge lexer supports both [apib and apiblueprint](https://github.com/jneen/rouge/pull/261/files#diff-61e0aed7ecc0f01100569d3a5a2e8e5cR8). I kind prefer the more verbose form.